packages/ag-charts-website/src/features/api-documentation/components/ApiReference.module.scss

```diff
@@ -159,34 +174,62 @@
     line-break: anywhere;
 }
 
-.metaList {
+.metaItem {
     display: flex;
+    flex-direction: row;
+    gap: $spacing-size-3;
+    align-items: center;
     flex-wrap: wrap;
-    gap: $spacing-size-2 $spacing-size-6;
-
-    h1 + & {
-        margin-top: 16px;
-    }
 }
 
-.metaItem {
+.metaRow {
     display: flex;
-    flex-direction: column;
+    flex-direction: row;
+    align-items: center;
 }
 
 .metaLabel {
-    font-size: var(--text-fs-xs);
+    font-size: var(--text-fs-sm);
     color: var(--color-fg-secondary);
+    opacity: 0.8;
 }
 
 .metaValue {
     font-size: var(--text-fs-sm);
     font-family: var(--text-monospace-font-family);
     color: var(--color-fg-code);
+    background-color: color-mix(in srgb, var(--color-fg-code), var(--color-bg-primary) 94%);
+    padding-top: 2px;
+    padding-bottom: 2px;
+    padding-left: 8px;
+    padding-right: 8px;
+    border-radius: var(--radius-sm);
+}
+
+.defaultValue {
+    font-size: var(--text-fs-sm);
+    padding: 2px 8px;
+    font-family: var(--text-monospace-font-family);
+    white-space: normal;
+    border-radius: var(--radius-sm);
+    background-color: var(--color-bg-code) !important;
+    color: var(--color-fg-primary);
+    font-size: var(--text-fs-sm);
+    font-variant-ligatures: none;
+    font-optical-sizing: auto;
+    font-weight: 400;
 }
 
+.initialItem,
 a.metaValue {
     color: var(--color-link);
+    text-decoration: underline;
+    padding-top: 2px;
+    padding-bottom: 2px;
+    padding-left: 8px;
+    padding-right: 8px;
+    border-radius: var(--radius-sm);
+    background-color: color-mix(in srgb, var(--color-link), var(--color-bg-primary) 90%);
 
     &:hover {
         color: var(--color-link-hover);
```

Comment:
Looks like a lot of shared styles between `.metaValue`, `.defaultValue`, `initialValue`. Wonder if we can refactor this to combine shared styles.

Reply:
👍🏼 Have moved shared styles to `import styles from '@ag-website-shared/components/reference-documentation/ApiReference.module.scss';` - will need another pass to make sure it's properly aligned between grid/charts